cmd/startnode.go

```diff
@@ -189,6 +189,7 @@ func init() {
 
 	StartnodeCmd.Flags().IntVar(&flags.MaxNonStakerPendingMsgs, "max-non-staker-pending-msgs", flags.MaxNonStakerPendingMsgs, "Maximum number of messages a non-staker is allowed to have pending. Defaults to `20`")
 
+	StartnodeCmd.Flags().StringVar(&flags.NetworkTimeoutReduction, "network-timeout-reduction", flags.NetworkTimeoutReduction, "Reduction of the timeout after a successful request, in nanoseconds. Defaults to `12ms`")
```

Comment:
Could you change the message as well to reflect that it's a Duration variable instead of in nanoseconds?